modules/core/include/opencv2/core/hal/intrin_neon.hpp

```diff
@@ -1206,6 +1207,16 @@ inline v_float16x4 v_cvt_f16(const v_float32x4& a)
 }
 #endif
 
+//! @name Check SIMD support
+//! @{
+//! @brief Check CPU capability of SIMD operation
+static inline bool hasSIMD128()
```

Comment:
I don't get the point.
NEON is not a part of instruction. It's an extention, so checking the hardware support has a meaning.
Why do we need to return ```true``` using macro?